src/bin/test_ioctls.rs

```diff
@@ -0,0 +1,38 @@
+use clap::{value_t, App, Arg};
```

Comment:
Are those also included when you install the crate? Maybe this could go to examples if that is the case?